app/assets/stylesheets/reviews.scss

```diff
@@ -1,3 +1,5 @@
+@import "variables";
```

Comment:
It has an error without import "variables"

![image](https://cloud.githubusercontent.com/assets/9666592/23342605/7878ea3c-fc2b-11e6-9191-f190118984fb.png)